src/common/util/global-scope.js

```diff
@@ -20,6 +20,8 @@ export let globalScope = (() => {
   throw new Error('New Relic browser agent shutting down due to error: Unable to locate global scope. This is possibly due to code redefining browser global variables like "self" and "window".')
 })()
 
+export let initialLocation = '' + globalScope.location
```

Comment:
I don't think we need the ability to set this variable anywhere, right?

```suggestion
export const initialLocation = '' + globalScope.location
```